action/init.go

```diff
@@ -42,6 +42,7 @@ const (
 	ETH_REDEEM               Type = 0x96
 	ERC20_LOCK               Type = 0x97
 	ERC20_REDEEM             Type = 0x98
+	ETH_REPORT_FAILED        Type = 0x99
```

Comment:
Decided not to use a seperate TX type , and using the refund flag in the existing ReportFinality mint struct.  Will rename the refund to a different identifier